addon/components/layout/left-menu/template.hbs

```diff
@@ -33,6 +33,20 @@
         {{/smart-link-to}}
       {{/active-link}}
     {{/if}}
+
+    {{#if hasPublishr}}
```

Comment:
can we split it in two?

* One link to publishrURL + `/campaigns`
* One link to publishrURL + `/payments`